### ISSUE TITLE
How to Load Image only from my profile ?

### ISSUE THREAD

**Issue by helmifluida** [konifar/material-cat#30]

**Comment by konifar** [comment]
@helmifluida Do you mention about profile icon in NavigationView?
Sorry, it is not implemented yet... because this app is for animation training.

**Comment by helmifluida** [comment]
Get item image From my profile Flickr ??

**Comment by konifar** [comment]
@helmifluida No. it is demo app. So I have no plan to develop login function now.
